io.github.gssh_linux.gssh.yml

```
@@ -0,0 +1,44 @@
+app-id: io.github.gssh_linux.gssh
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: runner.sh
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --env=GI_TYPELIB_PATH=/app/lib/girepository-1.0
+modules:
+  - name: Gssh
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 Flatpak/runner.sh /app/bin/runner.sh
+      - install -Dm755 main.py /app/bin/main.py
```

```suggestion
      - install -D main.py /app/bin/main.py
```

This isn't executable.